data/projects/antlr.md

```diff
@@ -0,0 +1,32 @@
+ANTLR (ANother Tool for Language Recognition) is a powerful parser generator
```

Comment:
I personally vote for this project to be taken to the wiki for now and removed from this page. I feel it might end up being similar to Redridge s project which nobody really used in the end.